app/Filament/Organizations/Resources/BeneficiaryResource/Pages/ViewBeneficiary.php

```diff
@@ -155,4 +169,107 @@ protected function personalInformationSection(): Section
                     ->label(__('field.has_protection_order')),
             ]);
     }
+
+    private function evaluations(): Group
+    {
+        return Group::make()
+            ->schema([
+                Section::make(__('beneficiary.page.create_initial_evaluation.title'))
+                    ->columnSpan(1)
+                    ->headerActions(
+                        [
+                            Action::make('edit')
+                                ->label(__('general.action.view_details'))
+                                ->url(fn ($record) => BeneficiaryResource::getUrl('create_initial_evaluation', ['record' => $record]))
+                                ->link()
+                                ->visible(fn ($record) => ! $record->violence?->violence_types),
+
+                            Action::make('view')
+                                ->label(__('general.action.view_details'))
+                                ->url(fn ($record) => BeneficiaryResource::getUrl('view_initial_evaluation', ['record' => $record]))
+                                ->link()
+                                ->visible(fn ($record) => $record->violence?->violence_types),
+                        ]
+                    )
+                    ->schema([
+                        Group::make()
+                            ->columns()
+                            ->schema([
+                                TextEntry::make('evaluateDetails.registered_date')
+                                    ->label(__('beneficiary.section.initial_evaluation.labels.registered_date'))
+                                    ->hidden(fn ($state) => $state == '-')
+                                    ->date(),
+                                TextEntry::make('violence.violence_types')
+                                    ->label(__('beneficiary.section.initial_evaluation.labels.violence_type'))
+                                    ->hidden(fn ($state) => $state == '-')
+                                    ->badge()
+                                    ->formatStateUsing(fn ($state) => $state != '-' ? $state->label() : ''),
+                                TextEntry::make('riskFactors.risk_level')
+                                    ->label('')
+                                    ->formatStateUsing(fn ($state) => $state != '-' ? $state->label() : '')
+                                    ->hidden(fn ($state) => $state == '-')
+                                    ->badge()
+                                    ->colors([
+                                        'success' => Level::LOW,
+                                        'warning' => Level::MEDIUM,
+                                        'danger' => Level::HIGH,
+                                    ]),
```

Comment:
Use coloros from enum  or like this if you have more enums with color prop : https://github.com/code4romania/asistent-medical-comunitar/blob/main/app/Concerns/Enums/HasColor.php

or just add a function color in the enum